docs/developer/app-store/apps/adyen.mdx

```diff
@@ -49,6 +49,10 @@ This section contains known limitations of this App.
 Saleor synchronous webhooks have a maximum response time limit of 20 seconds. The app restricts Adyen response time to 15 seconds to allow graceful error handling.
 If Adyen surpasses this limit, the App will return a FAILURE status with an appropriate error message (Timeout Error).
 
+### Maximum timeout for Saleor API calls is 5 seconds
+
+The apps restricts Saleor API response time to 5 seconds for [`TRANSACTION_INITIALIZE_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_initialize_sessionbcode) sync webhook. If Saleor API surpasses this limit, the App will gracefully continue processing.
+
```

Comment:
suggestion: I would add more info what is the result of such timeout, for example:

```suggestion
If such timeout happens, the created `TransactionItem` will not have the metadata with Adyen `additionalDetails`, that include payment method type, credit card brand, etc.
```